code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -974,7 +974,7 @@
 	blood_overlay_type = "armor"
 	body_parts_covered = CHEST
 	resistance_flags = NONE
-	armor = list("tier" = 2, "energy" = 0, "bomb" = 5, "bio" = 0, "rad" = 0, "acid" = -15) //nylon sucks against acid
+	armor = list("tier" = 2, "linemelee" = 5, "linebullet" = 5, "bomb" = 20, "bio" = 0, "rad" = 0, "acid" = -15) //nylon sucks against acid
```

Comment:
I'm actually not familiar with how the linemelee and linebullet values added like this works. Do they add to the tier presets? So this is melee 25 and brute 25 tier 2? If so it's fine, if its tier 5 melee rating it's not. Clarification needed.

Reply:
as it was explained to me on discord, line values are added on top of the base value
T-60 has linelaser = 25 and it is not likely for T10 armor to have only 25 laser protection
which seems to be the case if this comment is anything to go by "//If an armor has a tier value, having a linemelee value will add onto that tier value rather than replace it."

Reply:
Gotcha, that clears it up.